index.html

```diff
@@ -0,0 +1,248 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Admin Dashboard</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700;900&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div class="main-container">
+        <div class="header">
+            <div class="search-area">
+                <div class="search">
+                    <img class="icon" src="./icons/magnify.svg" alt="Search Icon" >
+                    <input type="text">
+                </div>
+                <div class="user-info">
+                    <img class="icon" src="./icons/bell-ring-outline.svg" alt="Notification Icon">
+                    <img src="./images/profile.png" alt="Profile Icon">
+                    <div class="user-name">Morgan Oakley</div></div>
+            </div>
+            <div class="header-info">
+                <div class="user-greeting">
+                    <img src="./images/profile.png" alt="User Icon">
+                    <div class="header-text">
+                        <div class="greeting">Hi there,</div>
+                        <div class="user-name">Morgan Oakley (@morgan)</div>
+                    </div>
+                </div>
+                <div class="btns">
+                    <button>New</button>
+                    <button>Upload</button>
+                    <button>Share</button>
+                </div>
+            </div>
+        </div>
+        <div class="sidebar">
+            <div class="dashboard-container">
+                <div class="dashboard-content">
+                    <img src="./icons/view-dashboard.svg" alt="Dashboard Icon">
+                    <div class="dashboard-text">
+                        Dashboard
+                    </div>
+                </div>
+            </div>
+            <div class="user-menus">
+                <div class="child">
```

Comment:
can the naming be little more relevant to the context, as discussed

Reply:
changed it to relevant name